src/models/Scholarships.test.ts

```diff
@@ -110,7 +117,12 @@ test('converter.fromFirestore', () => {
     {}
   );
 
-  expect(got).toEqual({ ...snapdata, deadline: new Date('2019-02-20') });
+  expect(got).toEqual({
+    ...snapdata,
+    deadline: new Date('2019-02-20'),
+    dateAdded: got.dateAdded,
+    lastModified: got.lastModified,
```

Comment:
Definitely want to use the dates from above (probably stored as a variable) cuz otherwise this only checks that these are what they already are.